Patch by Zarafa <http://www.zarafa.com/> - early catches an exception of vmime when iconv was unable to
convert a word from or to the requested charsets. This makes broken mails with invalid characters for a
valid charset still be able to deliver. We'd rather have an email with a ? sign than a FallbackDelivery
for the user. 

--- libvmime-0.7.1/src/word.cpp						2005-06-13 12:56:52.000000000 +0200
+++ libvmime-0.7.1/src/word.cpp.charset-catch				2005-11-15 18:02:45.000000000 +0100
@@ -688,7 +688,13 @@
 {
 	string out;
 
+	try {
 	charset::convert(m_buffer, out, m_charset, dest);
+	}
+	catch (vmime::exception &e) {
+		// copy 'word' as text
+		out = m_buffer;
+	}
 
 	return (out);
 }
